### DISCUSSION TITLE
There are no man pages for binaries e.g. clang in my llvm installation

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#1167] [Linux] [answered]
I have installed llvm on an Ubuntu machine through `brew install llvm` and there are no man pages for the binaries i.e. clang etc. included in the llvm installation.

**Comment by carlocab** [comment, marked as answer]
We don't build the documentation on purpose. You can always read them online.

**Reply by ghost** [reply]
What packages do you not build docs for?
Is there a list somewhere because I can access the man pages for some programs

**Reply by carlocab** [reply]
> What packages do you not build docs for?

The ones that require additional dependencies and build complexity to do so. `llvm` needs Sphinx or Doxygen to build the docs, so we skip those. Most other packages will include docs when you do `make install`, so there's no need to avoid building them.

> Is there a list somewhere because I can access the man pages for some programs

Not really, no. Maintaining such a list for ~5000 formulae is not something that's worth the effort unless one finds it enjoyable. And I'm not sure I know anyone who does, so I don't think there is one.
